packages/components/pie-textarea/src/index.ts

```diff
@@ -23,21 +26,44 @@ export class PieTextarea extends RtlMixin(LitElement) implements TextareaProps {
 
     @property({ type: String })
     @validPropertyValues(componentSelector, sizes, defaultProps.size)
-    public size?: TextareaProps['size'] = defaultProps.size;
+    public size = defaultProps.size;
+
```

Comment:
I wonder if you added `disabled` prop to `defaultProps`, maybe you can describe `disable` prop in  the same way?

```
@property({ type: Boolean, reflect: true })
public disabled = defaultProps.disabled;
```

Reply:
Ah yes, I must have missed that. Good idea.